server/conn_stmt.go

```diff
@@ -151,9 +151,14 @@ func (cc *clientConn) handleStmtExecute(data []byte) (err error) {
 			paramTypes = data[pos : pos+(numParams<<1)]
 			pos += (numParams << 1)
 			paramValues = data[pos:]
+			// Just the first StmtExecute packet contain parameters type,
+			// we need save it for further use.
+			stmt.SetParamsType(paramTypes)
```

Comment:
The parameters can be sent separately.
Say there are two params 'a' and 'b', 'a' is sent first, then 'b' is sent, if we set paramTypes like this, the type of  param 'a' would be lost.